src/common/iop_order.c

```diff
@@ -52,353 +53,289 @@
+                                  //    Really versatile yet under-used module, doing linear ops,
+                                  //    very good in scene-referred workflow
+  { {40.0f }, "basicadj", 0},        // module mixing view/model/control at once, usage should be discouraged
+  { {41.0f }, "colorbalance", 0},    // scene-referred color manipulation
+  { {42.0f }, "rgbcurve", 0},        // really versatile way to edit colour in scene-referred and display-referred workflow
+  { {43.0f }, "rgblevels", 0},       // same
+  { {44.0f }, "basecurve", 0},       // conversion from scene-referred to display referred, reverse-engineered
+                                  //    on camera JPEG default look
+  { {45.0f }, "filmic", 0},          // same, but different (parametric) approach
+  { {46.0f }, "filmicrgb", 0},       // same, upgraded
+  { {47.0f }, "colisa", 0},          // edit contrast while damaging colour
+  { {48.0f }, "tonecurve", 0},       // same
+  { {49.0f }, "levels", 0},          // same
+  { {50.0f }, "shadhi", 0},          // same
+  { {51.0f }, "zonesystem", 0},      // same
+  { {52.0f }, "globaltonemap", 0},   // same
+  { {53.0f }, "relight", 0},         // flatten local contrast while pretending do add lightness
+  { {54.0f }, "bilat", 0},           // improve clarity/local contrast after all the bad things we have done
+                                  //    to it with tonemapping
+  { {55.0f }, "colorcorrection", 0}, // now that the colours have been damaged by contrast manipulations,
+                                  // try to recover them - global adjustment of white balance for shadows and highlights
+  { {56.0f }, "colorcontrast", 0},   // adjust chrominance globally
+  { {57.0f }, "velvia", 0},          // same
+  { {58.0f }, "vibrance", 0},        // same, but more subtle
+  { {60.0f }, "colorzones", 0},      // same, but locally
+  { {61.0f }, "bloom", 0},           // creative module
+  { {62.0f }, "colorize", 0},        // creative module
+  { {63.0f }, "lowlight", 0},        // creative module
+  { {64.0f }, "monochrome", 0},      // creative module
+  { {65.0f }, "grain", 0},           // creative module
+  { {66.0f }, "soften", 0},          // creative module
+  { {67.0f }, "splittoning", 0},     // creative module
+  { {68.0f }, "vignette", 0},        // creative module
+  { {69.0f }, "colorreconstruct", 0},// try to salvage blown areas before ICC intents in LittleCMS2 do things with them.
+  { {70.0f }, "colorout", 0},
+  { {71.0f }, "clahe", 0},
+  { {72.0f }, "finalscale", 0},
+  { {73.0f }, "overexposed", 0},
+  { {74.0f }, "rawoverexposed", 0},
+  { {75.0f }, "dither", 0},
+  { {76.0f }, "borders", 0},
+  { {77.0f }, "watermark", 0},
+  { {78.0f }, "gamma", 0},
+  { { 0.0f }, "", 0 }
+};
+
+static void *_dup_iop_order_entry(const void *src, gpointer data);
+static int _count_entries_operation(GList *e_list, const char *operation);
+
+#if 0
```

Comment:
No, this routine is needed to instead a new module. But at the moment it is not used and so I commented it out to avoid an error from the compiler saying that the routine is not used.